demos/advanced-src/index.js

```diff
@@ -4,14 +4,28 @@ import '@babel/polyfill';
 
 import $ from 'jquery';
 
-import clspUtils from '~root/src/js/utils/utils';
-import IovCollection from '~root/src/js/iov/IovCollection';
-import TourController from '~root/src/js/iov/TourController';
+// simulate `import '@skylineos/clsp-player'`
+import {
+  IovCollection,
+  TourController,
+  utils as clspUtils,
+} from '~root/dist/clsp-player.min.js';
+
+/**
```

Comment:
i left it in there so that it's easy to test the require usage without creating an additional demo (for now, at least).  these demos are internal testing tools and example code - they aren't in and of themselves consumable products.  i'll delete it if you really care that much, but i'd prefer to leave it in for manual testing.

Reply:
@brainthinks Is this the only thing in the code left in for manual testing?

I think then we should write a quick jest test for this and delete the code.

Reply:
i asked to be allowed to put jest in to this project and i was told that it's too far outside of scope

Reply:
Discuss with @iankoplowitzdev  and Allyson to determine next steps for this issue.